superduperdb/db/base/backends.py

```diff
@@ -7,17 +9,23 @@
 from superduperdb.vector_search.inmemory import InMemoryVectorDatabase
 from superduperdb.vector_search.lancedb_client import LanceVectorIndex
 
-data_backends = {'mongodb': MongoDataBackend}
+DATA_BACKENDS = {'mongodb': MongoDataBackend}
 
-artifact_stores = {'mongodb': MongoArtifactStore}
+ARTIFACT_STORES = {'mongodb': MongoArtifactStore}
 
-metadata_stores = {'mongodb': MongoMetaDataStore}
+METADATA_STORES = {'mongodb': MongoMetaDataStore}
 
-vector_database_stores = {
+VECTOR_DATA_STORES = {
     config.LanceDB: LanceVectorIndex,
     config.InMemory: InMemoryVectorDatabase,
 }
 
-connections = {
+
+@functools.wraps(pymongo.MongoClient)
```

Comment:
I do not think we should do this. Wrapping like this has downsides (see [here](https://hynek.me/articles/decorators/) for example). Probably the downsides aren't an issue for most users, but it  will have impacts for some.

But I may have missed an important use-case here. What do we get from doing this?